### ISSUE TITLE
Comparing two columns Column_A greater than Column_B is not properly

### ISSUE THREAD

**Issue by ganeshtiger** [fivetran/great_expectations#2247]
**I am trying to compare two columns my expectation is column_A should be always greater than column_B.**

I have written my expectation like this,
 ```
 {
      "expectation_type": "expect_column_pair_values_A_to_be_greater_than_B",
      "kwargs": {
        "allow_cross_type_comparisons": true,
        "column_A": "updated_date",
        "column_B": "created_date",
        "ignore_row_if": "either_value_is_missing",
        "or_equal": false
      },
      "meta": {}
    },
```
After  running the expectations
In data docs, I am getting this error in status column,

`expect_column_pair_values_A_to_be_greater_than_B(**{'allow_cross_type_comparisons': True, 'column_A': 'product_wss_eula_date', 'column_B': 'product_wss_install_date', 'ignore_row_if': 'either_value_is_missing', 'or_equal': False, 'result_format': {'result_format': 'SUMMARY'}})`

I have uploaded the image FYR

![image (4)](https://user-images.githubusercontent.com/52994976/104083935-b152c480-5268-11eb-93d5-56cd90fe6fbb.png)

Any help on this will be very useful for me.

**I am using the Version: 0.13.2**

Thanks.

**Comment by eugmandel** [comment]
Thank you for filing this, @ganeshtiger! This seems to be an issue in the Data Docs rendering logic introduced in 0.13. The JSON of the expectation is rendered instead of its HTML representation. We will triage this week.

**Comment by eugmandel** [comment]
This is a duplicate of #2234 - closing. The core team is going to provide a fix.

**Comment by Quinten-gs** [comment]
@eugmandel This is currently still an issue in great expectations 0.13.13. Has this bugfix been fixed in the new version (0.13.14)? 

Many thanks for the amazing work on this module!
